### DISCUSSION TITLE
Welcome to OPNpool Discussions!

### DISCUSSION THREAD

**Discussion by cvonk** [cvonk/OPNpool#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hartmms** [comment]
Nice project!
  
I have a Pentair variable speed pump and an Intelli-chlor SWG. However, I didn't want to spend the $$$$ on a controller, so I have a particle photon that is connected to the RS485 bus to control the pump and SWG.  I also have a waterfall that can be enabled by turning a valve with the pump running.  I also have a generic LED pool light that can change modes by toggling the power through a set of patterns.  And finally, I haven i2c ORP and PH probes setup with a peristaltic pump to inject muratic acid based upon the Ph level. 

I have several switches setup in home assistant that allow for the pool light, pump and waterfall to be turned on and off.  I further have the PH and ORP levels graphed.  Finally, I have a text sensor that reads the remaining acid level in the tank.

**Comment by cvonk** [comment]
The house came with the pool equipment, so depending how you look at it, it
was free or very pricey.

**Comment by jaroy** [comment]
Wow, impressive effort! Curious if you have a rough estimate of the hours you've put into it?

I also bought a house with an existing Pentair system. Some recent time off led to a bunch of custom code running on an old Beaglebone. I spent a couple of days on it and didn't get as far as you before getting distracted. Eager to get rid of it and replace with OPNpool.

**Comment by cvonk** [comment]
This was not a project that I tracked hours for. It started just to trigger the kids' interest in embedded system. Introduced them to hardware toys like a scope, and a simple logic analyzer and moved onto more advanced debugger such as JTAG.
